src/main/java/com/epam/webdriver/WebDriverFactory.java

```diff
@@ -21,6 +22,8 @@ public static WebDriverFactory getDriverFactory() {
             return new ChromeDriverFactory();
         } else if (equalsIgnoreCase(browser, FIREFOX)) {
             return new FirefoxDriverFactory();
+        } else if (equalsIgnoreCase(browser, "sauceLabs")) {
```

Comment:
I suppose there is should be 2 flags to manage run.
1. Where to run: local, SauceLab or other
2.  What browser to use: Chrome, Firefox and other